src/Models.tsx

```diff
@@ -19,9 +20,14 @@ export type InBrainNativeSurveys = {
     value: number;
     currencySale: boolean;
     multiplier: number;
+    profileMatch: number;
```

Comment:
Hey @MusiikoAS - let's add const ProfileMatch (similar to const SurveyCategory) so the developers will be able so to compare integers to some readable value